src/model/UserphraseModel.cpp

```diff
@@ -194,23 +197,27 @@ void UserphraseModel::exportUserphrase(std::shared_ptr<UserphraseExporter> expor
     emit exportCompleted(result, exporter.get()->getPath(), exported);
 }
 
+QString UserphraseModel::checkBopomofo(const QString &bopomofo)
+{
+    QString replaceBopomofo = bopomofo;
+    QString qStr1 = QString::fromUtf8("ㄧ");
+    QString qStr2 = QString::fromUtf8("ㄧ");
+    replaceBopomofo.replace(qStr1,qStr2);
```

Comment:
This can be more concise:
```c++
replaceBopomofo.replace(QString::fromUtf8("ㄧ"), QString::fromUtf8("ㄧ"));  // one is Traditional Chinese character, another is Bopomofo
```